src/pal/src/exception/seh.cpp

```diff
@@ -259,10 +259,10 @@ bool CatchHardwareExceptionHolder::IsEnabled()
 --*/
 
 #ifdef __llvm__
-__thread 
-#else // __llvm__
+__thread
+#else
```

Comment:
BTW: The __llvm__  ifdef is not needed here because of PAL is always building with LLVM. You can just remove it.